src/graphics/vulkan/vulkan_surface_context.cpp

```
@@ -116,12 +139,11 @@ namespace moth_graphics::graphics::vulkan {
         // allocator
         {
             VmaAllocatorCreateInfo allocatorCreateInfo{};
```

Clang (14.0.6)

[error] variable 'allocatorCreateInfo' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)